internal/commands/logs/teamcity.go

```diff
@@ -0,0 +1,26 @@
+package logs
+
+import (
+	"github.com/cirruslabs/echelon"
+	"github.com/cirruslabs/echelon/renderers"
+	"strings"
+)
+
+func NewTeamCityLogsRenderer(renderer *renderers.SimpleRenderer) echelon.LogRendered {
+	return &FoldableLogsRenderer{
+		delegate:          renderer,
+		startFoldTemplate: "##teamcity[blockOpened name='%s']",
+		endFoldTemplate:   "##teamcity[blockClosed name='%s']",
+		escapeFunc: func(s string) string {
+			replacer := strings.NewReplacer(
```

Comment:
You can initialize the replacer before return and pass `replacer.Replace` as `escapeFunc`.

Reply:
Indeed, see https://github.com/cirruslabs/cirrus-cli/pull/118/commits/1e8cd06b804809575ab557123a4465c90c4fc6a5.